app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingScreen.kt

```diff
@@ -329,6 +349,31 @@ private fun VaultItemListingDialogs(
             )
         }
 
+        is VaultItemListingState.DialogState.Fido2PinSetUpPrompt -> {
+            var pin by rememberSaveable { mutableStateOf("") }
```

Comment:
I thought we decided that pins should not be saved?

Reply:
Yeah, you're right - I was trying to make it the same as the `AccountSecurityScreen`, but you're right that it should be updated there as well!